### PR TITLE
doc: added telnyx doc link to index.md

### PR DESCRIPTION
**Pull request by BoseSj**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
    - added telnyx documentation [link ](https://github.com/novuhq/novu/blob/84682038213c26c526f577c3668e028f6da4936a/docs/docs/channels/sms/telnyx.md)in the [index.md](https://github.com/novuhq/novu/blob/84682038213c26c526f577c3668e028f6da4936a/docs/docs/channels/sms/index.md) .
- **Why was this change needed?** (You can also link to an open issue here)
   - As the added documentation of SMS providers should be enlisted in [index.md](https://github.com/novuhq/novu/blob/84682038213c26c526f577c3668e028f6da4936a/docs/docs/channels/sms/index.md)
- **Other information**:
_NA_

### CODE REVIEW

**Comment by BoseSj** [comment]
please review  @davidsoderberg

File: docs/docs/channels/sms/index.md [hunk 1]
```diff
@@ -7,3 +7,4 @@ To read a provider specific documentation:
 - [AWS SNS](/channels/sms/sns)
 - [SMS77](/channels/sms/SMS77)
 - [TWILIO](/channels/sms/twilio)
+- [TELNYX](https://github.com/novuhq/novu/blob/84682038213c26c526f577c3668e028f6da4936a/docs/docs/channels/sms/telnyx.md)
```
Comment (p-fernandez) [inline docs/docs/channels/sms/index.md:10]:
```suggestion
- [TELNYX](/channels/sms/telnyx.md)
```

**Review by p-fernandez — CHANGES_REQUESTED** [review]

File: docs/docs/channels/sms/index.md (hunk 1 above)
Comment (BoseSj) [reply docs/docs/channels/sms/index.md:10]: - I tried that, but in GitHub md the link is giving 404 
image
- Works well though for website. 

So I was confused on that matter.

**Review by p-fernandez — APPROVED** [review]
🌟

**Pull request merged**